python/xorbits/_mars/lib/nvutils.py

```diff
@@ -535,7 +535,6 @@ def get_handle_by_uuid(uuid: bytes) -> _nvmlDevice_t:
 
 
 def get_mig_mode(device: _nvmlDevice_t) -> Tuple[int, int]:
-    _init_nvml()
```

Comment:
Why removing this line?

Reply:
seems like a merging issue, fixed in the latest commit